app/views/katello/api/v2/products/base.json.rabl

```diff
@@ -7,13 +7,30 @@ extends 'katello/api/v2/common/org_reference'
 
 attributes :provider_id
 attributes :sync_plan_id
-attributes :sync_status
 attributes :sync_summary
 attributes :gpg_key_id
 attributes :redhat? => :redhat
 
 attributes :available_content => :available_content, :if => params[:include_available_content]
 
+node :sync_status do |product|
+  {
+    :id => product.sync_status[:id],
+    :product_id => product.sync_status[:product_id],
+    :progress => product.sync_status[:progress],
+    :sync_id => product.sync_status[:sync_id],
+    :state => product.sync_status[:state],
+    :raw_state => product.sync_status[:raw_state],
+    :start_time => product.sync_status[:start_time],
+    :finish_time => product.sync_status[:finish_time],
+    :duration => product.sync_status[:duration],
+    :display_size => product.sync_status[:display_size],
+    :size => product.sync_status[:size],
+    :is_running => product.sync_status[:is_running],
+    :error_details => product.sync_status[:error_details]
+  }
+end
```

Comment:
You might simply add a local variable:
```
status = product.sync_status 
```

right before the hash, would simplify the hash quite a bit